Makefile

```diff
@@ -42,6 +42,10 @@ $(plugins_dir):
 test:
 	go test ./...
 
+lint:
+	# Version used should stay in sync with version in CI (.github/workflows/test.yaml).
+	docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:v1.50.1 golangci-lint run
```

Comment:
nit: docker is a dependency fort this make script. Even though it's easy to spot that a not installed docker is the culprit of the failing script, we could output a more user-friendly error here when docker is missing locally.

Reply:
I think we'll leave it for now and update later if necessary.